### ISSUE TITLE
[Bug]: PHB 2024 Artisan tool proficiency missing

### ISSUE THREAD

**Issue by johnweland** [foundryvtt/foundryvtt-premium-content#776]
### What happened?

So when creating a character using the 2024 PHB module. I can choose the Artisan background and choose tool proficiencies. I choose, say Jeweler's Tools and the blue dot fills in showing proficiency, however hovering gives a tool tip saying "not proficient"

![Screenshot 2024-09-28 at 5 52 18 PM](https://github.com/user-attachments/assets/f203d2fd-50af-411b-9922-5386e971b97a)
 

### What package are you reporting this for?

Dungeons & Dragons Player's Handbook

### Version

1.2.0

**Comment by mattexdee** [comment]
I haven't been able to reproduce this issue on my end. Is it still occurring for you in the latest version of the D&D5E game system? Here's the results I'm seeing:
<img width="782" alt="image" src="https://github.com/user-attachments/assets/24b852ce-4dbc-4839-abe9-7bd48d0b0fce" />

**Comment by johnweland** [comment]
Yeah so I made sure I've upgraded everything. I notice if I hover over the "dot" it says proficient. But when I hover the word "Int" or the tool name,  the popover has a pill/label at the bottom "Intelligence", "not equipped", "not proficient"

**Comment by krbz999** [comment]
This is a tooltip that displays the item in the compendium from which the options of proficiencies are derived.

Since the actor does not have *that* item equipped is why it displays like that.

https://github.com/foundryvtt/dnd5e/issues/3134
